res/CircleSchema/0.7/circle_schema.fbs

```diff
@@ -1431,6 +1433,12 @@ table GeluOptions {
   approximate: bool;
 }
 
+table CirGruOptions {
```

Comment:
Can you move this above `BCQGatherOptions` ?

Reply:
Done